ui/src/components/cTable/cTableHeader.vue

```diff
@@ -21,7 +21,7 @@ component.column.c-table-header(
             div.c-table-header__text {{ text }}
             c-icon.c-table-header__icon(
               :icon="icons[masterIcon]",
-              :class="{ 'c-table-header__icon_active': isIconActive }",
+              class="c-table-header__icon_active",
```

Comment:
What does this change do?

Reply:
basically, I removed it since that computed was not in the file anymore. So in order to avoid error, I removed that condition.
Anyways, we should do a huge clean up, and remove pagination and sorting. wdyt about removing it all at once? not now, not in this pr?

Reply:
lgtm